systemservices/marketplace/gen-schema

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+./node_modules/typescript-json-schema/bin/typescript-json-schema \
+  -o src/types/schema/manifest.json \
+  --required \
+  src/types/manifest.ts Manifest
```

Comment:
Can you move it inside the package.json in scripts array? Like this you don't need to specify the long path in the node_modules
```
typescript-json-schema -o src/types/schema/manifest.json --required src/types/manifest.ts Manifest
```

Reply:
done